config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
 
+  post 'review/check'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.